components/Link/src/index.scss

```diff
@@ -147,3 +147,9 @@ li .denhaag-link__icon {
   clip: rect(0, 0, 0, 0);
   border: 0;
 }
+
+p > .denhaag-link--with-icon-end.denhaag-link--external,
+.denhaag-unordered-list li > .denhaag-link--with-icon-end.denhaag-link--external,
+.denhaag-ordered-list li > .denhaag-link--with-icon-end.denhaag-link--external {
+  display: inline;
```

Comment:
Ik had dit eerder op display: inline-flex; gezet zodat er d.m.v. `gap` de spacing kon worden beheerd en de order met CSS aangepast kon worden.  Is dat komen te vervallen?

Zie dat nog wel op regel https://github.com/nl-design-system/denhaag/pull/1363/commits/d2828448b7857afa1692b571ed8468b61a9e42fd#diff-f5dd077880309a0f683854e1bc895188a027cbbb9992777f394845a8ce5254ddR70

Reply:
Uit screenshot hierboven gaat er nog wat mis met de link in de conditions. Daar heeft hij geen spacing?

Reply:
@paulAcato gefixt in deze commit: https://github.com/nl-design-system/denhaag/pull/1363/commits/f18deebe829169b2e7bc758fc673cd52f8418e75